tests/dragonfly/replication_test.py

```diff
@@ -317,6 +318,22 @@ async def start_master():
 
 
 
+"""
+Test key expiration. Test propagation from both eviction on access and garbage collection.
+"""
+
+expiration_cases = [
+    (100)
+]
+
+
+@pytest.mark.asyncio
```

Comment:
your rebase of main branch was not successful here
the flush all test here in mix with your test